nixpkgs/nixpkgs.bzl

```diff
@@ -56,12 +56,21 @@ nixpkgs_local_repository = repository_rule(
     },
 )
 
+def _is_supported_platform(repository_ctx):
+    if repository_ctx.execute(["nix-build","--version"]).return_code == 0:
+      nix_in_path = True
+    else:
+      nix_in_path = False
+    is_windows = repository_ctx.os.name.startswith("windows")
```

Comment:
Can we do the windows check before the shellout and shortcut if it’s windows?

Reply:
The whole windows check has been removed in https://github.com/tweag/rules_nixpkgs/pull/64